### DISCUSSION TITLE
Read individual table from geopackage

### DISCUSSION THREAD

**Discussion by om2468** [duckdb/duckdb#13263] [Q&A] [answered]
I am trying to read on table from a geopackage but the only SQL syntax I can find is select * from geopackage. How do I refer to a single table/feature class only. Many Thanks

**Comment by Maxxen** [comment, marked as answer]
Hello! You can use the `st_read()` function to provide additional options, such as the layer name. E.g. 

```sql
SELECT * FROM st_read('my_geopackage.gpkg', layer='my_layer_name');
```
